eth/beacon/state_machines/forks/serenity/configs.py

```diff
@@ -0,0 +1,16 @@
+
+from eth_utils import denoms
+from eth.beacon.state_machines.configs import BeaconConfig
+
+
+SERENITY_CONFIG = BeaconConfig(
+    BASE_REWARD_QUOTIENT=2**15,
+    DEFAULT_END_DYNASTY=9999999999999999999,
+    DEPOSIT_SIZE=32 * denoms.ether,  # WEI
+    CYCLE_LENGTH=64,  # slots
+    MIN_COMMITTEE_SIZE=128,  # validators
+    MIN_DYNASTY_LENGTH=256,  # slots
+    SHARD_COUNT=1024,  # shards
+    SLOT_DURATION=8,  # seconds
```

Comment:
Should that rather be named `SLOT_DURATION_SECONDS` and `SQRT_E_DROP_SECONDS`? Alternatively, instead of assigning `int` here, we could assign `Second` which would be a `NewType` and prevent arbitrary `int` values to be assigned. /cc @pipermerriam @carver

Reply:
Renaming: now these constants (configs) names follow the previous spec - and the spec is changing. Recently Justin proposed that using slot as the common time unit, except for `SLOT_DURATION`.  e.g., `SQRT_E_DROP_TIME` in second became `SQRT_E_DROP_TIME` in slot. I will clean up all these recent changes after porting old codebase.

For beacon chain, some other `NewType` candidates: `Slot`, `ShardID`. And [with the latest spec](https://github.com/ethereum/eth2.0-specs/blob/cbd254a3090771e79ccc8eb8e21c7b32dba93c45/specs/beacon-chain.md#constants): `ValidatorStatus`, `SpecialRecordType`, `ValidatorSetDeltaFlag`.

Reply:
#1383